iceoryx_dds/include/iceoryx_dds/dds/dds_config.hpp

```diff
@@ -24,7 +24,7 @@ namespace dds
 static constexpr units::Duration DISCOVERY_PERIOD = 1000_ms;
 static constexpr units::Duration FORWARDING_PERIOD = 50_ms;
 static constexpr uint32_t SUBSCRIBER_CACHE_SIZE = 128u;
-static constexpr uint32_t MAX_CHANNEL_NUMBER = iox::MAX_PORT_NUMBER;
+static constexpr uint32_t MAX_CHANNEL_NUMBER = iox::MAX_PUBLISHERS + iox::MAX_SUBSCRIBERS;
```

Comment:
@ithier This was a rough guess, I was not sure if number of publisher, number of subscribers or all of them together is the best fit here

Reply:
This is correct :smile:.